modules/monitoring-common-terms.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-overview.adoc
+
+:_content-type: REFERENCE
+[id="openshift-monitoring-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+alerts::
+
+Alertmanager::
+The Alertmanager service handles alerts received from Prometheus. Alertmanager is also responsible for sending the alerts to external notification systems.
+
+Alerting rules::
+Alerting rules contain a set of conditions that outline a particular state within a cluster. Alerts are triggered when those conditions are true. An alerting rule can be assigned a severity that defines how the alerts are routed.
+
+Cluster Monitoring Operator::
+The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
```

Comment:
The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus Operator which manages Prometheus and Alert manager, the Thanos Querier, the Telemeter Client, and Prometheus Adapter and ensures that they are up to date.